app/security_update_check.go

```diff
@@ -88,7 +89,11 @@ func (s *Server) DoSecurityUpdateCheck() {
 
 		defer res.Body.Close()
 
-		bulletins := model.SecurityBulletinsFromJson(res.Body)
+		var bulletins model.SecurityBulletins
+		if jsonErr := json.NewDecoder(res.Body).Decode(&bulletins); jsonErr != nil {
+			mlog.Error("Failed to decode JSON", mlog.Err(jsonErr))
```

Comment:
Gentle reminder to check our logging [principles](https://developers.mattermost.com/contribute/server/style-guide/#log-levels) before merging this change.